### DISCUSSION TITLE
Continuous use of the disk in idle mode

### DISCUSSION THREAD

**Discussion by StoneEngineer** [mediacms-io/mediacms#910] [Q&A]
MediaCMS is continuously using disks... no transcoding, no video playing, just idle run.
FYI: the arrangement is 2 mirrored nvme ssds serving as persistant volume for MediaCMS on Docker, thus two graphs.
Please advice.
![2023-11-09_13-53-03-614](https://github.com/mediacms-io/mediacms/assets/60100420/111d6b31-5318-45c7-8644-cc5befc756d8)

**Comment by mgogoulos** [comment]
What installation did you make, single server or Docker Compose? Are you sure there is no traffic on that time? Check the logs (tail -f logs/* for single server, or docker-compose logs -f for Docker Compose) at a time that there is no traffic, maybe there is something that runs (like a recuring task?)

**Reply by StoneEngineer** [reply]
Hi mgogoulos,
Thx for advice. This was docker swarm stack. However couldn't find anything suspicious in logs.
For now I created a dedicated VM for MediaCMS server and everything is working fine.
Will try docker swarm stack again in future.
